server/index.js

```diff
@@ -1,35 +1,69 @@
 
-
 const express = require('express');
+
 const server = express();
 const bodyParser = require('body-parser');
+const seed = require('../database/connection.js');
+
 const port = 3000;
-const listing = require('../database/index.js')
-const faker = require('../faker/faker.js')
-const pexel = require('../api/pexelKey')
+const listing = require('../database/index.js');
 
 
+const router = require('../routes/image-upload.js');
 
-server.use(bodyParser());
-
-server.use(express.static('public'))
+const upload = require('../api/s3.js');
+const request = require('request');
+const connection = require('../database/connection.js')
 
+server.use(bodyParser());
+server.use('/api', router);
+server.use(express.static('public'));
 server.listen(port, () => console.log(`server listening on port : ${port}`));
 
-server.get('/data', (req,res) => {
+server.get('/data', (req, res) => {
+  connection.connection.query('select * from images', (err, result) => {
```

Comment:
Why is connection written twice?

Reply:
thats the difference so the red is the past and i mad a space after comma that is after req

Reply:
I was referring to line 24 `connection.connection.query`, was this intentional or did you mean `connection.query`?